wp-modules/pattern-preview-renderer/pattern-preview-renderer.php

```diff
@@ -20,13 +20,13 @@
  * Receive pattern id in the URL and display its content. Useful for pattern previews and thumbnails.
  */
 function display_block_pattern_preview() {
-
-	// Nonce not required as the user is not taking any action here.
-	if ( ! isset( $_GET['pm_pattern_preview'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	if ( ! isset( $_GET['pm_preview_pattern_name'] ) ) {
 		return;
 	}
 
-	$pattern_name = sanitize_text_field( wp_unslash( $_GET['pm_pattern_preview'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	check_admin_referer( 'pm-pattern-preview' );
```

Comment:
Requiring a nonce for the pattern preview is pedantic. 

But it might prevent an argument.